spec/integration/delivery_spec.rb

```diff
@@ -94,14 +104,14 @@ def subscribe
     subscribe
     client.created('cats', 'https://example.com/cats/2')
     client.created('cats', 'https://example.com/cats/3')
-    processes.watch.wait_log %r(counter:events.added:2.*queue:demo)
+    processes.watch.wait_log %r(counter:events.added:2.*queue:#{uuid})
```

Comment:
It is the security key, but it's not new in this PR, it's just that the key changed.